exam_marker/Dockerfile

```diff
@@ -1,5 +1,28 @@
-FROM golang:1.17
-MAINTAINER Christian Heusel <christian@heusel.eu>
+FROM golang:1.17.6-bullseye
+# Specify pdfium version
+ARG PdfiumVersion=4861
+RUN DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get -yqq update
+RUN DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get -y install tzdata
+# Install pkg-config, etc.
+RUN  apt-get clean && apt-get install -yqq apt-utils pkg-config  wget
```

Comment:
The first two lines look like they can be omitted, or is `tzdata` really needed to do other installations?
Also please make this a single `RUN` statement in any case

Reply:
:heavy_check_mark: